src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Metrics/InstrumentAggregation.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Diagnostics.Metrics
+{
+    /// <summary>
+    /// This describes the type of aggregation we want to do for this instrument
+    /// </summary>
+    /// <remarks>
+    /// Right now we only support "temporal aggregation" - combining the values from multiple measurements taken at different
+    /// times with the same set of labels
+    /// In the future we could also support "spatial aggregation" - combining the values from multiple measurements that have
+    /// different sets of labels
+    /// </remarks>
+    internal class InstrumentAggregation
+    {
+        public InstrumentAggregation(MeasurementAggregation measurementAggregation)
+        {
+            MeasurementAggregation = measurementAggregation;
+        }
+        public MeasurementAggregation MeasurementAggregation { get; }
+    }
+
+    internal static class MeasurementAggregations
+    {
+        public static RateAggregation Rate = new RateAggregation();
```

Comment:
Can these be `static readonly`?
And use the short form `new()` as above?

Reply:
I deleted them during some refactoring (but if I had needed to keep them then yep, readonly would have worked)